### PR TITLE
Csv import

### PR DESCRIPTION
**Pull request by spencern**
Finishes CSV Product Import functionality

### CODE REVIEW

**Comment by spencern** [comment]
@paulgrever this one's bigger than it should be, probably should have broken it up into multiple commits, but works.

File: common/common.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    _.each(colorWords, function (word) {
+      colorSection = colorSection + word[0];
+    });
+  } else {
+    colorSection = colorWords[0].substr(0, 2).toUpperCase();
+  }
+
+  const sku = prodSection + '-' + colorSection + '-' + ImportProducts.letterSize(size);
+  return sku;
+};
```
Comment (paulgrever) [inline common/common.js:72]: @spencern is this how anthony wants the SKU?

File: client/templates/dashboard/dashboard.js [hunk 2]
```diff
@@ -39,5 +42,14 @@ Template.dashboardShopifyProducts.events({
...
       Meteor.call('importShopifyProducts/importProducts', update, productType);
     }
+  },
+  'submit #import-products-csv-form': function (event) {
+    event.preventDefault();
+    Papa.parse(event.target.csvImportProductsFile.files[0], {
+      header: true,
+      complete: function (results) {
+        Meteor.call('importCSVProducts/importProducts', results.data);
+      }
```
Comment (paulgrever) [inline client/templates/dashboard/dashboard.js:52]: Does papa parse require you to have an error pathway too? Or just a success?

File: common/common.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+ImportProducts.keyify = function (string) {
+  let keyifiedString = string.replace(/([\W\/])/ig, '');
+  keyifiedString = keyifiedString[0].toLowerCase() + keyifiedString.substr(1);
+  return keyifiedString;
+};
+
+ImportProducts.handleize = function (string) {
+  let handleizeString = string.replace(/([\W\/])/ig, '-');
+  return handleizeString.toLowerCase();
```
Comment (paulgrever) [inline common/common.js:32]: Regex = 👿

File: server/methods/importProducts.js [hunk 4]
```diff
@@ -61,10 +12,6 @@ function generateFakeLocation() {
   return _.sample(rooms) + '-S' + _.sample(shelves) + '-L' + _.sample(levels) + '-S' + _.sample(sections);
 }
```
Comment (paulgrever) [inline server/methods/importProducts.js:14]: Is this just until we get real room assignments?

**Comment by paulgrever** [comment]
Looks good.. I had higher level questions then code quality. Merging.

File: server/methods/importProducts.js (hunk 4 above)
Comment (spencern) [inline server/methods/importProducts.js:14]: Correct. Just to generate room samples for testing.

File: common/common.js (hunk 3 above)
Comment (spencern) [inline common/common.js:32]: love me some regex.

File: client/templates/dashboard/dashboard.js (hunk 2 above)
Comment (spencern) [inline client/templates/dashboard/dashboard.js:52]: Good catch. I should add an error handler in here. There's 2 ways to do it in Papa Parse - with an `error:` option and with an `error` object on the `results object` - I should add both.

Added issue #9 to deal with this.

File: common/common.js (hunk 1 above)
Comment (spencern) [inline common/common.js:72]: Not sure yet. We chatted about it and he's going to let us know what the formula is for his skus. We will probably generate all skus though, so this is a placeholder function until we know what his formula is.

**Pull request merged**